KamerConnect.DataAccess.Postgres/Repositorys/PersonRepository.cs

```diff
@@ -202,4 +202,63 @@ private static T ValidateEnum<T>(string value) where T : struct
 
         throw new ArgumentException($"Invalid value '{value}' for enum {typeof(T).Name}");
     }
+    
+    public void UpdatePerson(Guid personId, string fieldNameForIdCheck, List<string> fieldsToUpdate, List<NpgsqlParameter> paramaterNotations, string tableName)
```

Comment:
Dit is een manier om over het algemeen een table te updaten, maar het staat in de UpdatePerson. Bij een updatePerson functie zou ik verwachten een Person mee te geven aan de functie en dat hij in de database bij de desbetreffende persoon de data aanpast naar de Model die je meegeeft. Ik ben benieuwd hoe jij hierover denkt.